frequency-chain/docs
# update with new colors

Problem
=======
Theme adjustments for MD book to reflect current mocks

Closes #81 

Steps to Verify:
----------------
```shell
git fetch && git checkout feat/update-colors
npm install
npm run serve
```
1. Verify that the overall look is like the Figma mockups in the linked issue.
2. Make sure the SVGs have a matching background
3. This is less certain, but I also changed appearances of some links to try to match the mockups, which don't have links.  This is likely going to change so based on @wilwade's input it's less important given that it's in a branch.

Screenshots (optional):
-----------------------
![colors](https://github.com/user-attachments/assets/09577d9f-97ef-4d40-9c16-a8b6991315cb)
